styles/styles.css

```diff
@@ -15,6 +15,32 @@ header {
   box-shadow: 15px -15px #be7f6a;
 }
 
+.card1,
+.question,
+.answer {
+  background-color: #87adcf;
+  text-align: center;
+  margin: 3px;
+  padding: 1rem;
+}
+
+.answer {
+  color: #f3ead9;
+  background-color: #29282a;
+  border: 2px solid #e6664e;
+  padding: 1rem;
```

Comment:
And here you are overwriting `background-color` and setting the same `padding: 1rem` as above: please refactor to follow the DRY principle (Don't repeat yourself).

Reply:
sam as above; if I delete `padding: 1rem` here, it doesn't inherit from anywhere and looks different. Why is it bad to overwrite the `background-color `if I want another color for the button? What could be a better way?